src/main/java/org/profit/telegrambot/MaksymenBot.java

```diff
@@ -0,0 +1,169 @@
+
+
+public class MaksymenBot extends TelegramLongPollingBot {
+    @Override
+    public String getBotToken() {
+        return "5533026532:AAHGftCTJqPz0KKdegvqwUWUTUg69eDpfjU";
+    }
+
+    @Override
+    public String getBotUsername() {
+        return "@MaksymZymynTestBot";
+    }
+
+    private final ButtonService buttonService = new ButtonService();
+
+    @Override
+    public void onUpdateReceived(Update update) {
+        if (update.hasMessage() && update.getMessage().hasText()) {
+            if (update.getMessage().getText() != null) {
+                Message newMessage = update.getMessage();
+                String textMsg = newMessage.getText();
+                String chatId = String.valueOf(newMessage.getChatId());
+
+                if (textMsg.equals("/start")) {
+                    SendMessage sendMessage = new SendMessage();
+                    sendMessage.setText("Добридень, ми з України!\n" +
+                            "Введіть, будь ласка, своє ім'я та прізвище:");
+                    sendMessage.enableMarkdown(true);
+                    sendMessage.setChatId(chatId);
+                    try {
+                        execute(sendMessage);
+                    } catch (TelegramApiException e) {
+                        e.printStackTrace();
+                    }
+
+                } else if (textMsg.equals("Хліби")) {
+                    /*
+                    Для добавления одной кнопки:
+                    String textNewMsg = "Оберіть, який саме хліб Ви хочете замовити:";
+                    keyboard(chatId, textNewMsg, "Бородінський");
+                     */
+                    String textBread = "Оберіть, який саме хліб Ви хочете замовити:";
+                    sendInlineKeyboard(textBread, chatId, "Бородінський", "Василівський");
+                    // "Литовський", "Ризький", "Заріченський"
+                } else if (textMsg.equals("Торти")) {
+                    String textCake = "Оберіть, який саме торт Ви хочете замовити:";
+                    sendInlineKeyboard(textCake, chatId,  "Київський", "Мадонна");
+                    // "Сакура", "Йогуртовий", "Деметра"
+                } else if (textMsg.equals("Тістечка")) {
+                    String textCupcake = "Оберіть, яке саме тістечко Ви хочете замовити:";
+                    sendInlineKeyboard(textCupcake, chatId,  "Еклер", "Лимонне");
+                    // "Спартак",  "Корзинка", "Медове"
+                } else if (textMsg.equals("Круасани")) {
+                    String textPie = "Оберіть, який саме круасан Ви хочете замовити:";
+                    sendInlineKeyboard(textPie, chatId,  "З вишнею", "Зі згущеним молоком");
+                    // "З малиною", "З шоколадом", "Зі смородиною"
+                } else {
+                        String textNewMsg = textMsg + ", ми пропонуємо:";
+                        sendReplayKeyboard(textNewMsg, chatId);
+                    }
+                }
+            }
+        }
+
+
+
```

Comment:
Could you please go over the PR double-checking the blank lines? 

I would recommend keeping a single line between methods and fields declarations.